gateway/meta.yaml

```diff
@@ -1,5 +1,7 @@
 { "display_name":"SignalFx Gateway",
-  "description": "Aggregate metrics from different sources and formats and forward them to SignalFx.",
+  "description": "Aggregate metrics from different sources and formats and forward them to SignalFx. 
+  
+  If you are an APM customer, install Smart Gateway instead.",
```

Comment:
```suggestion
  If you are an APM customer, install the Smart Gateway instead.",
```

Reply:
Also delete the comma at end of sentence